dev/io.openliberty.microprofile.telemetry.1.0.internal.jaeger_fat/fat/src/io/openliberty/microprofile/telemetry/internal/tests/AgentConfigTest.java

```diff
@@ -70,14 +72,19 @@
 @MaximumJavaLevel(javaLevel = 20)
 public class AgentConfigTest {
 
-    @Server("TelemetryAgentConfig")
+    private static final String SERVER_NAME = "TelemetryAgentConfig";
+
+    @Server(SERVER_NAME)
     public static LibertyServer server;
 
     @ClassRule
     public static JaegerContainer jaegerContainer = new JaegerContainer().withLogConsumer(new SimpleLogConsumer(JaegerBaseTest.class, "jaeger"));
 
+    // In contrast to most tests, this test needs a new jaeger instance for each repeat
+    // so that it can check for any trace IDs not accounted for
+
     @ClassRule
-    public static RepeatTests r = MicroProfileActions.repeat("spanTestServer", MicroProfileActions.MP61, MicroProfileActions.MP60);
+    public static RepeatTests r = FATSuite.allMPRepeats(SERVER_NAME);
```

Comment:
Is this comment correct I don't see anything that's checking for traces not accounted for?

Either way, you should probably use a `RuleChain` to ensure the rules execute in the right order.